test/sagemaker_tests/mxnet/training/resources/hvdbasic/train_hvd_basic.py

```diff
@@ -0,0 +1,11 @@
+import json
```

Comment:
The Apache License header seems to have been applied inconsistently across these scripts. Some of them have the header, and some don't.

@nskool Do you know if the license in the LICENSE file applies across the whole repository?

Reply:
As per previous directive, LICENSE file does apply across the whole repository. However, having it as part of individual files is good-to-have. So please update the license if possible for these files.

Reply:
Sure. adding.